src/featherlight.js

```diff
@@ -66,31 +66,31 @@
 					'sandbox', 'src', 'srcdoc', 'width'],
 				attrs = {};
 
-			whiteList.map(function(item) {
+			$.each(whiteList, function(index, item) {
 				var attrValue = parseAttrs(obj, 'iframe')[item];
```

Comment:
Just noticed, but would be better to call `parseAttrs` once instead of once per item in whiteList...